views/reducers/session.jsx

```diff
@@ -15,13 +16,22 @@ const user = function user(state = {
         isProcessing: false,
         signedIn: true,
         email: action.email,
+        authError: false,
       });
     case SIGNED_OUT:
       return Object.assign({}, state, {
         isProcessing: false,
         signedIn: false,
         email: '',
       });
+    case ERROR:
```

Comment:
Error is a bit ambiguous. It should match closer to RESET_AUTH_ERROR like XXXX_AUTH_ERROR